crates/cheatnet/src/forking/state.rs

```diff
@@ -80,11 +81,13 @@ impl BlockInfoReader for ForkStateReader {
             Ok(MaybePendingBlockWithTxHashes::Block(block)) => {
                 let block_info = BlockInfo {
                     block_number: BlockNumber(block.block_number),
-                    timestamp: BlockTimestamp(block.timestamp),
                     sequencer_address: block.sequencer_address.into_(),
+                    block_timestamp: BlockTimestamp(block.timestamp),
+                    gas_prices: ForgeGasPrices::default().into(),
+                    use_kzg_da: false,
```

Comment:
I think the comment explaining this abbreviation would be great 🙏

Reply:
[Add information about EIP-4844](https://github.com/foundry-rs/starknet-foundry/pull/1738/commits/f2a49e96cb1117cbac9033ea4dee97f54f007b76)